src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlRetry/SqlRetryServiceOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage
+{
+    /// <summary>
+    /// Provides SQL retry parameters Used to initialize SqlRetryService.
+    /// </summary>
+    public class SqlRetryServiceOptions
+    {
+        /// <summary>Used to bind this class to .NRT options data source.</summary>
+        public const string SqlServer = "SqlServer";
```

Comment:
Low priority suggestion: you can use "SqlServer = nameof(SqlServer);"

Reply:
This is supposed to be a name of a section in a json file. Shouldn't really be tied to any class. Microsoft recommends this pattern for IOptions initialization: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-7.0 ,  Also went through all of our startup code, every place uses same pattern: .GetSection(plain-string).Bind(class-instance);

But I found something else that needs to be resolved once FhirServer is fully transitioned to the new retry logic. I left a big TODO comment in the code. Keep in mind that this PR is just a partial transition to the new retries, there's still work to be done, but we needed to get export functionality transitioned first.